runtimes/peregrine/src/tests.rs

```diff
@@ -30,6 +38,58 @@ fn call_size() {
 	);
 }
 
+#[test]
+fn attestation_storage_sizes() {
+	let attestation_record = attestation::AttestationDetails::<Runtime>::max_encoded_len();
+	let delegation_record = BoundedVec::<
+		<Runtime as frame_system::Config>::Hash,
```

Comment:
S: These values should be probably split up into different variables first. The same for the rest below and in the other runtimes.

Reply:
Having a type for it might be nice but I would not split this up.